ee/ucp/interlock/usage/service-clusters.md

```diff
@@ -4,205 +4,257 @@ description: Learn how to route traffic to different proxies using a service clu
 keywords: ucp, interlock, load balancing, routing
 ---
 
-## Configure Proxy Services
-With the node labels, you can re-configure the Interlock Proxy services to be constrained to the
-workers for each region. For example, from a manager, run the following commands to pin the proxy services to the ingress workers:
+Reconfiguring Interlock's proxy can take a long time (1-2 seconds) per overlay network managed by that proxy. In order to scale up to larger number of Docker networks and services routed to by Interlock, you may consider implementing *service clusters*: multiple proxy services managed by Interlock (rather than the default single proxy service), each responsible for routing to a separate set of Docker services and their corresponding networks, thereby minimizing proxy reconfiguration time.
```

Comment:
A couple things. The last sentence in this paragraph is a bit of a run on. Could you break it apart?

Also, stylistically we try to keep each line to 80 characters. Not a big issue, but it helps with reviewing stuff.